hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/replication/HiveSyncGlobalCommitParams.java

```diff
@@ -85,7 +85,9 @@ public void load() throws IOException {
 
   Properties mkGlobalHiveSyncProps(boolean forRemote) {
     TypedProperties props = new TypedProperties(loadedProps);
-    props.putAll(globalHiveSyncConfigParams.toProps());
+    globalHiveSyncConfigParams.toProps().entrySet().forEach(
+        e -> props.putIfAbsent(e.getKey(), e.getValue())
+    );
```

Comment:
why this change? if anything not working, it should be `globalHiveSyncConfigParams.toProps()` to be fixed, not making a workaround here, which masks the root issue.